main.rb

```diff
@@ -1,21 +1,61 @@
-require_relative 'lib/product.rb'
-require_relative 'lib/storage.rb'
-require_relative 'lib/warehouse.rb'
-require_relative 'lib/cart.rb'
+require 'bundler/setup'
+require 'sinatra/base'
 
-products = [Product.new("Milk", 2, 23), Product.new("Book", 10, 8), Product.new("Gems", 100, 23)]
+Dir['./lib/**/*.rb'].each { |file| require file }
 
-ware = Warehouse.new("Main", products)
+module Store
+  PRODUCTS = [
+    Product.new('Milk', 2, 23),
+    Product.new('Book', 10, 8),
+    Product.new('Gems', 100, 23)
+  ]
 
-ware.add(1, 1)
-ware.add(2, 2)
-ware.to_s
+  WAREHOUSE = []
+  CART = []
 
-cart = Cart.new("Shopping", ware)
-cart.add(1, 1)
-cart.add(1, 2)
-cart.add(3, 1)
-cart.to_s
+  class App < Sinatra::Base
```

Comment:
There is a convention to use path after namespace. In your case you have `App` inside `Store` module, so you should probably put it into `store/app.rb`